backend/app/controllers/spree/admin/payments_controller.rb

```diff
@@ -80,11 +80,11 @@ def object_params
 
       def load_data
         @amount = params[:amount] || load_order.total
-        @payment_methods = Spree::PaymentMethod.active.available_to_admin
+        @payment_methods = Spree::PaymentMethod.active.available_to_admin.ordered_by_position
         if @payment && @payment.payment_method
           @payment_method = @payment.payment_method
         else
-          @payment_method = @payment_methods.first
+          @payment_method = @payment_methods.ordered_by_position.first
```

Comment:
Hmm, this is already ordered by position, since it's using `@payment_methods` already. I believe it should be fine as it was.